### DISCUSSION TITLE
Installation of WebPageRecordingPlugin in new AWS instance failing

### DISCUSSION THREAD

**Discussion by dbwelch** [ant-media/Ant-Media-Server#5069] [Q&A]
ubuntu@ip-xxxxxxxxxx:~$ wget https://github.com/ant-media/Plugins/blob/addWebPageDevelopmentPlugin/WebpageRecordingPlugin/build/webpage-recording-plugin.jar
--2023-05-03 22:54:16--  https://github.com/ant-media/Plugins/blob/addWebPageDevelopmentPlugin/WebpageRecordingPlugin/build/webpage-recording-plugin.jar
Resolving github.com (github.com)... 140.82.114.4
Connecting to github.com (github.com)|140.82.114.4|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘webpage-recording-plugin.jar’

webpage-recording-plugin.jar                    [ <=>                                                                                     ] 135.47K  --.-KB/s    in 0.004s  

2023-05-03 22:54:16 (37.5 MB/s) - ‘webpage-recording-plugin.jar’ saved [138719]

ubuntu@ip-xxxxxxxxxx:~$ ls
webpage-recording-plugin.jar
ubuntu@ip-xxxxxxxxxx:~$ sudo cp webpage-recording-plugin.jar /usr/local/antmedia/plugins
ubuntu@ip-xxxxxxxxxx:~$ sudo service antmedia restart
ubuntu@ip-xxxxxxxxxx:~$ curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "http://localhost:5080/WebRTCAppEE/rest/webpage/record/start" -d '{"url": "http://example.com"}'
HTTP/1.1 404 
Access-Control-Allow-Origin: *
Content-Type: text/html;charset=utf-8
Content-Language: en
Content-Length: 713
Date: Wed, 03 May 2023 22:56:46 GMT

<!doctype html><html lang="en"><head><title>HTTP Status 404 – Not Found</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 404 – Not Found</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Message</b> Not Found</p><p><b>Description</b> The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.</p><hr class="line" /><h3>Apache Tomcat/8.5.82</ubuntu@ip-xxxxxxxxxx:~$ 

This is a newly created instance on AWS using the enterprise version, no changes made, nothing in logs indicating errors or installation of plugin.

Thank you!
David

**Comment by yashtandon113** [comment]
Hi @dbwelch 
Thanks for asking this question.

Because the web page recording plugin is not merged, you receive a 404 error when calling the Rest API. I will discuss it with the team regarding the status of this feature.
Here is the related PR: https://github.com/ant-media/Plugins/pull/46
